client/fs_endpoint_test.go

```diff
@@ -1897,6 +1899,15 @@ func TestFS_logsImpl_NoFollow(t *testing.T) {
 	case <-time.After(10 * time.Duration(testutil.TestMultiplier()) * streamBatchWindow):
 		t.Fatalf("did not receive data: got %q", string(received))
 	}
+
+	select {
```

Comment:
Eek, clicked too fast, should these cases be folded into the select above?

Reply:
I don't think so, as we want to be sure that something comes down both `resultCh` *and* `errCh`, not one or the other.

Reply:
Hm, but it appears resultCh is only closed if the expected data is received. If an error occurs, framer won't be closed, so the resultCh will block until its timeout is reached. Meanwhile the source of the failure is probably sitting in errCh and will never get received?

Reply:
I've reworked this to rid us of the `logsImpl()` goroutine, instead just running the code sequentially but with a one-second `context.WithTimeout()`.

Reply:
Great idea! Now we need to steal that idea for our other streaming endpoint tests. :sweat_smile:

Thanks for sticking with this and your other great work! We appreciate it!